packages/graphql/tests/tck/connections/filtering/interface-relationships.test.ts

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Neo4jGraphQL } from "../../../../src";
+import { formatCypher, formatParams, translateQuery } from "../../utils/tck-test-utils";
+
+describe("interface relationships with aliased fields", () => {
+    let typeDefs: string;
+    let neoSchema: Neo4jGraphQL;
+
+    beforeAll(() => {
+        typeDefs = /* GraphQL */ `
+            interface Production {
+                title: String!
+            }
+
+            type Movie implements Production {
+                title: String! @alias(property: "movieTitle")
+                runtime: Int!
+            }
+
+            type Series implements Production {
+                title: String! @alias(property: "seriesTitle")
+                episodes: Int!
+            }
+
+            type ActedIn @relationshipProperties {
+                screenTime: Int!
+            }
+
+            type Actor {
+                name: String!
+                currentlyActingIn: Production @relationship(type: "CURRENTLY_ACTING_IN", direction: OUT)
+                actedIn: [Production!]! @relationship(type: "ACTED_IN", direction: OUT, properties: "ActedIn")
+            }
+
+            type ProtectedActor
+                @authorization(
+                    validate: [{ where: { node: { actedInConnection: { node: { title: "$jwt.title" } } } } }]
```

Comment:
`actedInConnection` is deprecated it will be better starting using the new one as `actedInConnection_SOME` as it will be easier to deprecate later!
```suggestion
                    validate: [{ where: { node: { actedInConnection_SOME: { node: { title: "$jwt.title" } } } } }]
```